### DISCUSSION TITLE
SelectFile() Not working as expected in Chrome

### DISCUSSION THREAD

**Discussion by baguilar-work** [cypress-io/cypress#20274] [Questions and Help]
Ive been trying to use cy.selectFile() to upload a file in Cypress test. The following code looks as such:
```

cy.get('#new-project-photo', { force: true }).selectFile({
    contents: 'cypress/fixtures/media/003_StreetView.jpg',
    fileName: 'image01.jpg',
    mimeType: 'image/jpeg',
}, { force: true });
```

The code when running the test through FF runs as expected and the file is uploaded with a 200 on the post request. However when running the test through Chrome. I get a 400 with an error of `{"error":{"message":"Missing required parameter - file"}}` (Coming from server). The upload does work as expected when manually doing the upload in chrome.

The URL it sends the POST to is a different domain then the base url of the tested site. Could this be the cause? Im also curious about having this in the config:  ` "chromeWebSecurity": false`, Which is needed to load the site properly in chrome.

Ive also tried wrapping the image in a fixture first, same issue.

Anyone know what i might be doing wrong and why it only seems to work in FF?

**Comment by portyu9** [comment]
Hi @baguilar-work 

This was a Cypress version issue.

Upgrade from Cypress `9.5.0` to at least `9.5.1` (preferably the latest supported Cypress version):

```bash
npm install --save-dev cypress@latest
```

The same Chrome `selectFile()` problem was reported with 9.5.0, and the reporter confirmed that upgrading to 9.5.1 fixed the upload.

Your cross-domain Cloudinary endpoint is therefore not the primary issue.

Please mark this answer as accepted if it helped.
